content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -204,11 +222,18 @@ By querying the series of statements, consumers can verify who did what and when
 1. Query DataTrails for the collection of statements
 
     ```bash
-    PARAMS="event_attributes.subject=${SUBJECT}&page_size=3"
+    PARAMS="event_attributes.subject=${SUBJECT}&page_size=1"
```

Comment:
When running the samples multiple times, a user could wind up with a lot of statements for the subject "my-product-id".  For this reason, I added page_size to 3 to simulate a collection, but not overwhelm the screen with a long list that scrolls past their previous commands.

This PR changes it to 1. Was this intentional?

Reply:
changing back to 3
```suggestion
    PARAMS="event_attributes.subject=${SUBJECT}&page_size=3"
```